src/DGmethods/NumericalFluxes.jl

```diff
@@ -35,6 +38,7 @@ struct CentralGradPenalty <: GradNumericalPenalty end
 function diffusive_penalty!(::CentralGradPenalty, bl::BalanceLaw,
                             VF, nM, diffM, QM, aM, diffP, QP, aP, t)
   FT = eltype(QM)
+  deriv_types = unit_scale(vars_gradient(bl,FT), inv(space_unit(bl)))
```

Comment:
grad is probably better than deriv
```suggestion
  grad_types = unit_scale(vars_gradient(bl,FT), inv(space_unit(bl)))
```